kolibri/plugins/facility/views.py

```diff
@@ -109,19 +106,16 @@ def exported_csv_info(request, facility_id):
             else:
                 start = ""
                 end = ""
-            log_path = os.path.join(
-                logs_dir,
-                CSV_EXPORT_FILENAMES[log_type].format(
+                filename = CSV_EXPORT_FILENAMES[log_type].format(
                     facility.name, facility.id[:4], start[:10], end[:10]
-                ),
-            )
+                )
         else:
-            log_path = os.path.join(
-                logs_dir,
-                CSV_EXPORT_FILENAMES[log_type].format(facility.name, facility.id[:4]),
+            filename = CSV_EXPORT_FILENAMES[log_type].format(
+                facility.name, facility.id[:4]
             )
-        if os.path.exists(log_path):
-            csv_statuses[log_type] = os.path.getmtime(log_path)
+
+        if default_storage.exists(filename):
```

Comment:
According to Sentry there's a path whereby this variable doesn't exist yet: https://learningequality.sentry.io/issues/6321649781/?alert_rule_id=14557147&alert_type=issue&notification_uuid=f911561f-142a-43f1-b240-99174fc0f77d&project=1378215&referrer=slack

Reply:
Actually had a call with @nucleogenesis earlier about his access challenges; the `django-storages` library suggests using [oath2 creds for use with creds when passing settings to the library](https://django-storages.readthedocs.io/en/latest/backends/gcloud.html#settings) like so:
```
from google.oauth2 import service_account

GS_CREDENTIALS = service_account.Credentials.from_service_account_file(
    "path/to/credentials.json"
)
```
But i'd like to try just using [Application Default Credentials](https://googleapis.github.io/google-api-python-client/docs/oauth-server.html#application-default-credentials); I think that might work across local dev and cloud envs
```
import google.auth
credentials, project = google.auth.default()
```